package.json

```diff
@@ -0,0 +1,83 @@
+{
+    "name": "@bancor/dad-bridge-wrapper",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/bancorprotocol/dad-bridge-wrapper.git"
+    },
+    "homepage": "https://github.com/bancorprotocol/dad-bridge-wrapper#readme",
+    "bugs": {
+        "url": "https://github.com/bancorprotocol/dad-bridge-wrapper/issues"
+    },
+    "author": "Barak Manos",
+    "license": "SEE LICENSE IN LICENSE",
```

Comment:
can simply be MIT here as well

Reply:
Can you please give me the exact version?

Reply:
I'm not sure but it's the one that was in the repo originally (set by github)

Reply:
It's the `package.json` file here. GitHub doesn't include this file by default as far as I know (it wasn't in the repo that you've created, I just copied it from the V3 repo).

Reply:
it should just say "MIT"

Reply:
Fixed.